docs/source/user_guide/installation.md

```diff
@@ -1,79 +1,113 @@
 (target-installation)=
 # Installation
 
-## Install the package
-:::{admonition} Use a conda environment
-:class: note
-To avoid dependency conflicts with other packages, it is best practice to install Python packages within a virtual environment.
-We recommend using [conda](conda:) or [mamba](mamba:) to create and manage this environment, as they simplify the installation process.
-The following instructions assume that you have conda installed, but the same commands will also work with `mamba`/`micromamba`.
-:::
+## Use a virtual environment
 
-### Users
-To install movement in a new environment, follow one of the options below.
-We will use `movement-env` as the environment name, but you can choose any name you prefer.
+While not strictly required, it is highly recommended to install `movement` into a
+clean virtual environment, using tools such as [conda](conda:) or [uv](uv:).
+
+This should be set up before installing `movement`.
 
 ::::{tab-set}
-:::{tab-item} Conda
-To create an environment with the core package only:
+:::{tab-item} conda
+Create and activate a new environment:
```

Comment:
We ask users to make sure they have uv installed and provide the link to the installation, but we don't for conda. Wdyt about replacing the urls in 
https://github.com/neuroinformatics-unit/movement/blob/98f429e2badaf871741a562a6a1e1fc150c2f6ab/docs/source/user_guide/installation.md?plain=1#L7
to point directly to the respective installation pages?

In the tab-items we can then simplify both conda and uv instructions to:
Create and activate a new virtual environment